tools/cldr-apps/js/src/esm/cldrTable.js

```diff
@@ -1301,12 +1321,37 @@ function getValidWinningValue(theRow) {
   return null;
 }
 
+function makeRowId(id) {
+  return ROW_ID_PREFIX + id;
+}
+
+function isHeaderId(id) {
+  return id.startsWith(HEADER_ID_PREFIX);
+}
+
+function makeHeaderId(name) {
+  // Use encodeURIComponent since, for example, name might
+  // be "Western Asia", and spaces are illegal in id attributes;
+  // add HEADER_ID_PREFIX to identify this kind of id uniquely
+  return HEADER_ID_PREFIX + encodeURIComponent(name);
```

Comment:
Better (more human friendly if the links are passed around) would probably be to strip out spaces, leaving `#WesternAsia`

Reply:
sounds good! I'll strip the spaces if there's no objection...

maybe stripping out (rather than escaping) some other punctuation would be even more human-friendly, however I'm not very confident the headers would be unique (in a page) if all punct were removed -- any thoughts on that?

Reply:
i think they probably are unique, but safest would be to have a test to confirm.  @macchiati  thoughts?

Reply:
Although I haven't done an exhaustive test that header_ ids are unique, I'm fairly confident; eventually we should have a mode for cldr-apps-webdriver to cycle through all pages and validate the html -- that would catch any non-unique id in a more general way